craft_cli/dispatcher.py

```diff
@@ -270,6 +271,22 @@ def _get_requested_help(self, parameters):
         help_text = self._help_builder.get_command_help(command, options)
         return help_text
 
+    def _build_no_command_error(self, missing_command: str) -> str:
+        """Build the error help text for missing command, providing options."""
+        all_alternatives = self.commands.keys()
+        similar = difflib.get_close_matches(missing_command, all_alternatives)
+        if len(similar) == 0:
+            extra_similar = ""
+        else:
+            if len(similar) == 1:
+                similar_text = repr(similar[0])
+            else:
+                *previous, last = similar
```

Comment:
Nice technique to separate the last element, I'll use it when opportunity arises.